cpp/arcticdb/entity/types.hpp

```diff
@@ -637,14 +637,14 @@ struct StreamDescriptor {
         util::variant_match(id,
             [this] (const StringId& str) { data_->set_str_id(str); },
             [this] (const NumericId& n) {
-                util::check(n >= 0, "Negative NumericId is not supported");
+                util::check(n >= 0, "Negative Numeric symbol is not supported");
```

Comment:
Minor: I would use a small "N"  to avoid any confusion with Numeric Investors.

The original capital "N" was part of a type name.